api/middleware/stats.go

```diff
@@ -82,5 +85,7 @@ func RequestStats() macaron.Handler {
 }
 
 func pathSlug(p string) string {
-	return strings.Replace(path.Clean(p), "/", "_", -1)
+	slug := strings.TrimSuffix(path.Clean(p), "/")
```

Comment:
once path.Clean is done, there is no trailing / (see https://golang.org/pkg/path/#Clean where you can try it)